dts/bindings/kscan/ite,it8xxx2-kscan.yaml

```diff
@@ -14,6 +14,15 @@ properties:
     interrupts:
       required: true
 
+    wucctrl-0:
```

Comment:
why is the `-0` index notation used? can there be `-N` entries?

Reply:
Done, remove -0 index.